basic-ellipsis/public/style/main.css

```diff
@@ -0,0 +1,324 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Lato;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 29px;
+  text-align: center;
+}
+
+.wrapper {
+  max-width: 1920px;
+  box-sizing: border-box;
+  margin: 0 auto;
+  padding: 75px 150px;
+  background-color: #ffffff;
+  border-color: #000000;
+}
+
+.wrapper:first-child {
+  padding: 0;
+  padding-top: 250px;
+}
+
+.wrapper:last-child {
+  padding: 0;
+  padding-bottom: 50px;
+}
+
+.wrapper__theme_services {
+  background-image: url('../images/ServicesBG.png');
+  background-position: no-repeat no-repeat;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.content_container {
+  max-width: 900px;
+  margin: 0 auto;
+  border: 2px solid;
+  white-space: pre-line;
+}
+
+.wrapper:first-child > .content_container {
+  padding: 0;
+  padding-top: 100px;
+}
+
+.content_container__without_border {
+  border: none;
+}
+
+.content_container__border_secondary_color {
+  border-color: #ffffff;
+}
+
+.content_container__text_secondary_color {
+  color: #ffffff;
+}
+
+.content_container__services {
+  border-top: none;
+}
+
+.content_container__services__header:before, .content_container__services__header:after {
+  white-space: nowrap;
+  content: "";
+  border-top: 2px solid white;
+  width: 50%;
+}
+
+.content_container__services__header > span {
```

Comment:
we should not use elements in BEM, also there is no place for element of element